src/app/shared/logout/logout.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from 'src/app/auth-service';
+@Component({
+  selector: 'app-logout',
+  templateUrl: './logout.component.html',
+  styleUrls: ['./logout.component.css']
+})
+export class LogoutComponent implements OnInit {
+
+  constructor(private authService: AuthService) { }
+
+  ngOnInit(): void {
```

Comment:
ngOnInit ?

Reply:
Removed unused ngOninit method. And pushed the changes.